p1-staking-pool-dyn/src/lib.rs

```diff
@@ -95,16 +96,23 @@ impl Contract {
         }
     }
 
-    /// Returns amount of staked NEAR, farmed CHEDDAR and the current round.
-    pub fn status(&self, account_id: AccountId) -> (U128, U128, u64) {
+    /// Returns amount of total staked NEAR, user staked NEAR, farmed CHEDDAR
+    /// and the current round.
+    pub fn status(&self, account_id: AccountId) -> (U128, U128, U128, u64) {
         let mut v = self.get_vault_or_default(&account_id);
         return (
+            self.total_stake.into(),
```

Comment:
This breaks the UI. 
This does not go here.
status is the account status, should not include contract information.

Reply:
yes, I made a note about it in the PR description. The change is motivated by the UI update. I wanted to reduce amount of calls to the network. Let's discuss on in chat to make a decision if we want this change or not.